### ISSUE TITLE
enableViewChange should be per-skipchain

### ISSUE THREAD

**Issue by jeffallen** [dedis/cothority#1474]
From skipchain.go:

```
// If this flag is set, then we relax our forward-link signature verification
// to accept the aggregate signature by the public keys of the rotated roster.
// View-change should only be enabled if there are no security implications
// when the forward-links are signed by the rotated roster instead of the
// roster in the skipblock that points to it. The security implication depends
// on the services that use skipchain, so it is disabled by default. The option
// can be changed from outside of the package by calling EnableViewChange. This
// flag should ideally be the service configuration, but other structs depend
// on it too, e.g., SkipBlock, so we keep it in a global.
var enableViewChange bool
var enableViewChangeOnce sync.Once
```
The task is to make this be a per-skipchain flag, and not a global for the entire service.

**Comment by Gilthoniel** [comment]
Closed by #1538

**Comment by Gilthoniel** [comment]
Closed by #1538

**Comment by Gilthoniel** [comment]
Closed by #1538

**Comment by jeffallen** [comment]
More correctly: #1538 is a better proposal for how to do this.
